safe_transaction_service/history/indexers/ethereum_indexer.py

```diff
@@ -162,10 +168,11 @@ def get_to_block_number(
         """
         :param from_block_number:
         :param current_block_number:
+
         :return: Top block number to process
         """
         return min(
-            from_block_number + self.block_process_limit,
+            from_block_number + self.block_process_limit - 1,
```

Comment:
I don't get why minus 1?

Reply:
No, minimum value for `block_process_limit` is `1`. `0` would be no limit but it cannot be used due to the RPC limits. So in this case imagine you want to process `1 block`, then `to and from block` will be the same